src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1528,12 +1541,12 @@ public override bool IsDBNull(int ordinal)
         /// <returns><b>true</b> if the specified column value is equivalent to <see cref="DBNull"/> otherwise <b>false</b>.</returns>
         public override async Task<bool> IsDBNullAsync(int ordinal, CancellationToken cancellationToken)
```

Comment:
Why doesn't `cancellationToken` have a default value?

Reply:
I will look into this (and the below) tomorrow.

Reply:
Seems I forgot about this for a while :)

When these async APIs were added to ADO.NET (back in .NET Framework 4.5), the pattern was to have two overloads: a non-virtual overload without a cancellation token delegating to a second, virtual overload with a cancellation token. We're overriding the second, which is why there's no default here. [You can see the two overloads here](https://docs.microsoft.com/en-us/dotnet/api/system.data.common.dbdatareader.isdbnullasync?view=netframework-4.8#System_Data_Common_DbDataReader_IsDBNullAsync_System_Int32_System_Threading_CancellationToken_).

Note that when adding new async APIs nowadays, that pattern is no longer used and we have just one overload with default.